client/state/data-layer/wpcom/me/two-step/index.js

```diff
@@ -0,0 +1,9 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import applicationPasswords from './application-passwords';
+import { mergeHandlers } from 'state/action-watchers/utils';
+
+export default mergeHandlers( applicationPasswords );
```

Comment:
I'm curious about this file, what are the motivations for adding the handler in this way? It seem like a lot of ceremony to add a handler simply because it's in a deeper directory.

Reply:
This is because we're trying to make data layer directories mirror the REST API endpoint route. Seemed like an overkill to me at first, but I'm starting to love it - it makes it so easy to find the relevant data layer code that corresponds to an endpoint.

Reply:
But is `mergeHandlers` necessary? Can it just `export default applicationPasswords`?

Reply:
No, `mergeHandlers` isn't necessary and can be removed. It just makes it easy to add new handlers to an existing file.

But if you insist, I can remove it 😉

Reply:
I'd never _insist_, but it is a bit of unnecessary overhead that is pretty trivial to add when (and if) it's needed. Up to you what to do 👍